schema205/json_translate.py

```diff
@@ -221,6 +221,8 @@ def _get_simple_constraints(self, constraints_str, target_dict):
                         target_dict[mx] = maximum
                     elif '%' in c:
                         target_dict['multipleOf'] = int(numerical_value)
+                    elif 'string' in target_dict['type']:
```

Comment:
@tanaya-bigladder also note the addition of the string pattern constraint. I needed to add this to validate that the `data_model` in all the examples was `"ASHRAE 205"`. Maybe there's a better way to handle this?